vignettes/estimate_time_varying_severity.Rmd

```diff
@@ -196,18 +122,18 @@ ggplot(df_covid_uk_subset) +
   )
 ```
 
-Then, we plot the death incidence data with following command:
+We also plot the death incidence data over time.
```

Comment:
I'm wondering about adding these sections that are about plotting the data prior to applying the functions, it's nice but maybe the content of the vignettes should go a bit more straight to the point of demonstrating the functionality of the package and the plotting that's not showcasing the outputs of the functions would fit better on a tutorial or learning platform like Applied Epi?

Reply:
Yes, point taken, and I'm not a big fan of these sections either. I'm going to keep them for now as they were added by Tim R, and if we think they should be removed in a full package review, I will do that.